dss/api/bundles/__init__.py

```diff
@@ -351,11 +352,22 @@ def _idempotent_save(blobstore: BlobStore, bucket: str, key: str, data: dict) ->
         return False, existing_data == data
     else:
         # write manifest to persistent store
-        blobstore.upload_file_handle(
-            bucket,
-            key,
-            io.BytesIO(json.dumps(data).encode("utf-8")),
-        )
+        _d = json.dumps(data).encode("utf-8")
+        size = len(_d)
+        part_size = 16 * 1024 * 1024
```

Comment:
Any reason `16 * 1024 * 1024` was chosen?

Reply:
16MB seemed to optimize upload performance.

Reply:
Sounds good!